src/FsAutoComplete/LspServers/FSharpLspClient.fs

```diff
@@ -143,74 +144,164 @@ type ServerProgressReport(lspClient: FSharpLspClient, ?token: ProgressToken) =
 
 open System.Diagnostics.Tracing
 open System.Collections.Concurrent
+open System.Diagnostics
+open Ionide.ProjInfo.Logging
+
+
+/// listener for the the events generated from the fsc ActivitySource
+type ProgressListener(lspClient: FSharpLspClient) =
+
+  let isOneOf list string = list |> List.exists (fun f -> f string)
+
+
+  let strContains (substring: string) (str: string) = str.Contains(substring)
+
+  let interestingActivities =
+    [
+
+      strContains "BoundModel."
+      strContains "IncrementalBuild."
+      strContains "CheckDeclarations."
+      strContains "ParseAndCheckInputs."
+      strContains "BackgroundCompiler."
+      strContains "IncrementalBuildSyntaxTree."
+      strContains "ParseAndCheckFile."
+      strContains "ParseAndCheckInputs."
+      strContains "CheckDeclarations." ]
```

Comment:
This list is a bit aggressive on what gets tracked.  Should probably review what we'd actually want to track.